gslib/tests/test_ls.py

```diff
@@ -395,6 +396,26 @@ def test_400_location_redirect(self):
   def test_301_location_redirect(self):
     self.location_redirect_test_helper('eu-west-1', 'us-east-2')
 
+  @SkipForJSON('Only the XML API supports changing the calling format.')
+  def test_default_gcs_calling_format_is_path_style(self):
+    bucket_uri = self.CreateBucket()
+    object_uri = self.CreateObject(bucket_uri=bucket_uri, contents=b'foo')
+
+    stderr = self.RunGsUtil(['-D', 'ls', suri(object_uri)], return_stdout=True)
+    self.assertIn('Host: storage.googleapis.com', stderr)
+
+  @SkipForJSON('Only the XML API supports changing the calling format.')
+  def test_gcs_calling_format_is_configurable(self):
+    bucket_uri = self.CreateBucket()
+    object_uri = self.CreateObject(bucket_uri=bucket_uri, contents=b'foo')
+
+    with SetBotoConfigForTest([('s3', 'calling_format',
+                                'boto.s3.connection.SubdomainCallingFormat')]):
```

Comment:
Should this more clearly be a custom value?

Reply:
Gave it a variable name!!